elasticsearch/src/main/java/com/gentics/mesh/search/index/node/NodeIndexHandlerImpl.java

```diff
@@ -633,4 +633,9 @@ public Stream<? extends HibNode> loadAllElements() {
 		return CommonTx.get().nodeDao().findAllGlobal();
 	}
 
+	@Override
+	public boolean isDefinitionDataDependent() {
```

Comment:
Here and below: there is a need to split the test context init into the data-unaware index init (comes before the data population) , and data-aware index init (after the data is populated).